lib/ui/widgets/list_product_card.dart

```diff
@@ -0,0 +1,70 @@
+import 'package:flutter/material.dart';
+import 'package:mini_project_1/shared/shared_value.dart';
+import 'package:mini_project_1/ui/widgets/product_card.dart';
+
+class ListProductCard extends StatelessWidget {
+  final Function(Map) onTap;
+  final double? width;
+  final double? height;
+  const ListProductCard(
+      {Key? key, this.width, this.height, required this.onTap})
+      : super(key: key);
+
+  @override
+  Widget build(BuildContext context) {
+    List<Map> data = [
+      {
+        'productName': 'Standard Drone Camera',
+        'productDesc': 'Package Weight: 235g',
+        'productRatting': '4.9',
+        'totalRatting': '542',
+        'productPrice': '86',
+        'image': 'assets/image 37.png',
+        'backgroundColor': CustomColors.backgroundImage1,
+      },
+      {
+        'productName': '3D flip Drone Camera',
+        'productDesc': 'Product weight: 70g',
+        'productRatting': '4.8',
+        'totalRatting': '497',
+        'productPrice': '78',
+        'image': 'assets/image 35.png',
+        'backgroundColor': CustomColors.backgroundImage2,
+      },
+      {
+        'productName': 'Remote Control Drone',
+        'productDesc': 'Wrist Remote Control',
+        'productRatting': '3.2',
+        'totalRatting': '21',
+        'productPrice': '43',
+        'image': 'assets/image 351.png',
+        'backgroundColor': CustomColors.backgroundImage3,
+      },
+    ];
+
+    return SizedBox(
+      width: width,
+      height: height,
+      child: ListView.builder(
+        itemCount: data.length,
+        itemBuilder: (BuildContext context, int index) {
+          return Padding(
+            padding: const EdgeInsets.symmetric(vertical: 8, horizontal: 24),
+            child: GestureDetector(
+              onTap: onTap(data[index]),
+              child: ProductCard(
+                productName: data[index]['productName'],
+                productDesc: data[index]['productDesc'],
+                productRatting: data[index]['productRatting'],
+                totalRatting: data[index]['totalRatting'],
+                productPrice: data[index]['productPrice'],
+                imageUrl: data[index]['image'],
+                color: data[index]['backgroundColor'],
```

Comment:
ini di assign di satu variable aja, `final product = data[index];` jadi pas di productCard tinggal panggil product['productName'] dll

Reply:
iya mas, rencana juga mau refactor ke model yang udah dibuat mas @anasmc97 di branch [[#2duje6t] task/Search and navbar](https://github.com/anasmc97/mini_project_majoo_1/pull/7/files#)